TicTacToeApp/GamePrompt.swift

```diff
@@ -1,29 +1,28 @@
 
+import UIKit
+
 public class GamePrompt {
+  var prompt: UILabel
 
-  public init() {
-    // initialize with the UILabel?
+  public init(prompt: UILabel) {
+    self.prompt = prompt
```

Comment:
If subclassing UIView wasn't so frustrating, I'd have been tempted to implement GamePrompt as a subclass of UILabel. As it is, it seems about equivalent to me whether GamePrompt wraps a UILabel vs. outputs text which is assigned to a UILabel externally.